src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/organisations/OrganisationsDpsApiService.kt

```diff
@@ -148,109 +162,6 @@ class OrganisationsDpsApiService(@Qualifier("organisationsDpsApiWebClient") priv
 }
 
 //  Fake DTOs - replace with real ones once created
-data class SyncCreateOrganisationRequest(
```

Comment:
Most fake ones now replaced; Organisation types and address phone still to be done